helm/prometheus-rules/templates/alerting-rules/capi-cluster.rules.yml

```diff
@@ -15,7 +15,7 @@ spec:
           labels:
```

Comment:
What about other valid phases like Provisioning, Running, Pending... ?

Reply:
All are valid phases as long as the cluster is not stuck on one of them for too long. The only phase that the cluster should be for long time is the `Provisioned` phase. So the alert will page if the cluster is for 1h on a phase that is not `Provisioned`.